uint32 drone_id

GPS position

bool insertBeforeNextWayPoint
bool standByAndReplanRouteAtPoint